File: Pythonleetcode/Leetcode88.py
# class Solution(object):
#     def merge(self, nums1, m, nums2, n):
#         k=0
#         for i in range(n):
#             for j in range(m+n):
#                 if nums2[i]<nums1[j]:
#                     nums1=nums1[:j]+[nums2[i]]+nums1[j:]
#                     k=k+1
#                     break
#
#         nums1=nums1[:m+k]+nums2[k:]
#         return nums1



class Solution(object):
    def merge(self, nums1, m, nums2, n):
        n1=nums1[:m]
        n2 = nums2[:n]
        n1=sorted(n1+n2)
        nums1[0:]=n1












print(Solution().merge([1,2,3,4,0,0,0],4, [2,5,6],3))

File: CppPractice/CppLeetcode/Leetcode88.cpp
class Solution {
public:
    void merge(vector<int>& nums1, int m, vector<int>& nums2, int n) {
        vector<int> saved;
        for(int i=0;i<m;i++){saved.push_back(nums1[i]);}
        for(int i=0;i<n;i++){saved.push_back(nums2[i]);}
        sort(saved.begin(),saved.end());
        for(int i=0;i<m+n;i++){nums1[i] = saved[i];}
        
    }
};